src/auth/registration/Register.tsx

```diff
@@ -0,0 +1,61 @@
+import { AuthContainer } from "auth/views/components/AuthContainer";
+import { Form, Formik } from "formik";
+import React, { useState } from "react";
+import { RegistrationSchema } from "auth/registration/RegistrationSchema";
+import { initialValues, RegistrationForm } from "auth/registration/types";
+import { Stack } from "@mui/material";
+import { NameField } from "auth/registration/NameField";
+import { StepsButton } from "auth/registration/StepsButton";
+import { AdvertiserField } from "auth/registration/AdvertiserField";
+import { AddressField } from "auth/registration/AddressField";
+import { FormikSubmitButton } from "form/FormikHelpers";
+import { useRegister } from "auth/hooks/mutations/useRegister";
+import { AdvertiserRegistered } from "auth/registration/AdvertiserRegistered";
+
+export function Register() {
+  const [step, setStep] = useState(0);
+  const { register, hasRegistered, error } = useRegister();
+
+  if (hasRegistered || error) {
+    return (
+      <AuthContainer>
+        <AdvertiserRegistered error={error} />
+      </AuthContainer>
+    );
+  }
+
+  return (
+    <AuthContainer height={step === 2 ? "650px" : "475px"}>
+      <Formik
```

Comment:
Perhaps consider using a [stepper](https://mui.com/material-ui/react-stepper/#linear) to give the user an overview of how many screens they're going to encounter?